04_setup_ironic.sh

```diff
@@ -159,6 +159,29 @@ if [ ! -f "${CACHED_MACHINE_OS_IMAGE}" ]; then
   pushd $(dirname ${CACHED_MACHINE_OS_IMAGE})
   sha256sum --strict --check ${CACHED_MACHINE_OS_IMAGE}.sha256sum || ( rm -f "${CACHED_MACHINE_OS_IMAGE}" ; exit 1 )
   popd
+  # If using the live images, fetch the kernel, rootfs and initramfs images also
+  if [ "${RHCOS_LIVE_IMAGES}"  == "true" ]; then
```

Comment:
This needs to be something like `"${RHCOS_LIVE_IMAGES:-false}"` to prevent an error if it is not defined in the config.
Or alternatively we could put `RHCOS_LIVE_IMAGES=${RHCOS_LIVE_IMAGES:-false}` in common.sh.